tests/rptest/remote_scripts/stream_verifier_txn.py

```diff
@@ -0,0 +1,1637 @@
+    # timestamp for last message
+    last_message_ts: float
+    # how much ms should pass between messages to be sent
+    msgs_rate_ms: float
+    # how much time to wait when waiting for the next message
+    consume_timeout_s: int
+    # Precreated Producer class.
+    producer: ck.Producer
+    # All data needed to create consumer class.
+    # As opposed to Producer, Consumer must be created/used in the same thread
+    consumer_config: dict
+    # EOF flag
+    reached_eof: bool
+    # Termination signalling flag
+    # It is simpler that threading event and faster
+    terminate: bool
+
+    @property
+    def transaction_id(self):
+        return f"{self.id}-{self.source_topic_name}-{self.last_message_ts}"
+
+
+class MessageTransforms:
+    """Class to hold message transforming functions
+    """
+    @staticmethod
+    def dezero_transform(src_key, src_value):
+        """Removes zeroes from all numbers in value
+        Example:
+            src value "aaa00023bbb_453z0002"
+            returned: "aaa23bbb_453z2"
+
+        Args:
+            src_key (str): message key
+            src_value (str): message value
+
+        Returns:
+            str, str: transformed key and value
+        """
+        active_number = ""
+        new_value = ""
+        for char in src_value:
+            if char in string.digits:
+                active_number += char
+            else:
+                new_int = int(active_number) if len(active_number) > 0 else 0
+                if new_int > 0:
+                    new_value += str(new_int) + char
+                    active_number = ""
+                else:
+                    new_value += char
+        if len(active_number) > 0:
+            new_int = int(active_number)
+            new_value += str(new_int)
+        return src_key, new_value
+
+    @staticmethod
+    def deserialize(value: bytes, ctx) -> str:
+        return value.decode("utf-8")
+
+
+class MessageGenerator:
+    """Various Generator functions to use in Producer
+    """
+    @staticmethod
+    def gen_indexed_messages(start_index: int,
+                             message_count: int) -> Generator:
+        """Generates indexed messages:
+        'key_0'/'0000', 'key_1'/'0001', ...
+
+        Args:
+            start_index (int): message index to start from
+            message_count (int): number of messages to generate
+
+        Yields:
+            Generator: message_key: int, message_value: int
+        """
+        for idx in range(start_index, start_index + message_count):
+            key = f"key_{idx}"
+            value = f"{idx:04}"
+            yield key, value
+
+
+class StreamVerifier():
+    """Main class to process messages
+    """
+    def __init__(self,
+                 brokers,
+                 workload_config: dict,
+                 worker_threads: int,
+                 rate=0,
+                 total_messages=100):
+        # Remove quotes if any
+        self.logger = setup_logger('core')
```

Comment:
For consistency, would probably be a good idea to turn `'core'` here into `LOGGER_CORE` and define that up with the other logger names, no?
```suggestion
        self.logger = setup_logger(LOGGER_CORE)
```

Reply:
Yes, thanks!